### PR TITLE
9 :arrow_right: main

### PR DESCRIPTION
**Pull request by Crola1702**
# ➡️ Forward port

Port 9 to main

Branch comparison: https://github.com/gazebosim/gz-fuel-tools/compare/main...9

**Note to maintainers**: Remember to **Merge** with commit (not squash-merge or rebase)

### CODE REVIEW

**Review by azeey — APPROVED** [review]

**Comment by azeey** [comment]
I can't merge because GitHub says "This branch cannot be rebased due to conflicts". Can you merge from `main` locally and push?

**Comment by Crola1702** [comment]
I don't see any new changes from main, so I don't have anything else to push here :thinking:

**Review by azeey — CHANGES_REQUESTED** [review]
I actually found a couple of issues. Maybe making those changes and pushing will fix the GitHub issue?

File: package.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>gz-fuel_tools9</name>
```
[the remark this answers is not in the document]
Comment (Blast545) [reply package.xml]: Addressed on: c991fc2

File: package.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  <buildtool_depend>cmake</buildtool_depend>
+
+  <build_depend>gz-cmake4</build_depend>
+
+  <depend>gz-common6</depend>
+  <depend>gz-math8</depend>
+  <depend>gz-msgs11</depend>
+  <depend>gz-tools2</depend>
+  <depend>gz-utils2</depend>
```
[the remark this answers is not in the document]
Comment (Blast545) [reply package.xml]: Addressed on: c991fc2

**Review by azeey — APPROVED** [review]

**Pull request merged**